examples/gcp/provider.tf

```diff
@@ -2,15 +2,15 @@
 provider "netapp-gcp" {
   # specify projectID or project number
   # alternatively, set GCP_PROJECT environment variable
-  # project         = "123456890"
+  project         = "cv-solution-architect-lab"
 
   # We support two ways to authenticate. Choose one or the other
 
   # Auth option 1: using service account key file
   # path to JSON key file for IAM service account with "roles/netappcloudvolumes.admin" privileges
   # alternatively, set GCP_SERVICE_ACCOUNT environment variable
 
-  # service_account = "/Users/abc/key.json"
+  service_account = "/Users/gpablo/Documents/cv-solution-architect-lab-2b441b722fef.json"
```

Comment:
I prefer to keep the original path example, but i am going to merge this PR. and undo this from bitbucket then push back to github.